nireports/tests/test_reportlets.py

```diff
@@ -258,6 +259,15 @@ def test_plot_melodic_components(tmp_path, outdir):
         out_file=no_classified,
     )
 
+    plot_melodic_components(
+        str(melodic_dir),
+        in_fname,
+        tr=2.0,
+        report_mask=report_fname,
+        out_file=no_classified_compress,
+        compress=True,
+    )
```

Comment:
This test now fails if `svgo` and `cwebp` are not available - you'll need to gate this section (or make a standalone test) similar to https://github.com/nipreps/nireports/blob/f51b36c964bb5321f3be2a65865e989a7e05ec30/nireports/tests/test_interfaces_utils.py#L35-L37

Reply:
I know (https://github.com/nipreps/nireports/pull/163#issuecomment-2638558657). Tests can be skipped for local testing of an alpha user that does not have the optional dependencies, but they should be tested by the CIs. Otherwise the added code is dead code: zero guarantee that it is working.

Reply:
I agree, but no need to test on both circle/gha, especially since this is an optional dependency.

Reply:
:+1: Correct, thanks for having elaborated a little bit more. See 6ac3353. GHA is still failing:
https://github.com/nipreps/nireports/actions/runs/13182911357/job/36797950771?pr=163#step:14:358

I was not able to make it work yesterday (https://github.com/nipreps/nireports/pull/163#issuecomment-2638558657).